specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/workspaceRP.json

```diff
@@ -3581,23 +3661,22 @@
       "type": "string",
       "x-ms-parameter-location": "method"
     },
+    "PaginationParameter": {
+      "in": "query",
+      "name": "$skip",
+      "description": "Continuation token for pagination.",
```

Comment:
>pagination [](http://example.com/codeflow?start=45&length=10)

security review: ensure it contains no sensitive data!

Reply:
Yes, this already exist in the path definition before, this PR is just to put in the parameter section.